src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -105,15 +105,33 @@ protected override bool IsParentAnAttribute(SyntaxNode node)
         {
             // If everything classifies the same way, then just pick that classification.
             using var _ = PooledHashSet<ClassifiedSpan>.GetInstance(out var set);
+            using var _s = PooledHashSet<ClassifiedSpan>.GetInstance(out var sset);
```

Comment:
1. we woudl name these `_1` and _2`.  
2. the comment `// If everything classifies the same way, then just pick that classification.` is not correct anymore.
3. `sset` isn't a good name.